README.md

```diff
@@ -7,7 +7,7 @@ Flight Control is a service for declarative, GitOps-driven management of edge de
 ## Building
 
 Prerequisites:
-* `git`, `make`, and `go` (>= 1.21), `openssl`, `openssl-devel`, and `podman-compose`
+* `git`, `make`, and `go` (>= 1.21), `openssl`, `openssl-devel`, `kind`, `kubectl` and `podman-compose`
```

Comment:
these are test requirements can we add them to the test README. The product itself does not rely on these.

https://github.com/flightctl/flightctl/blob/main/test/README.md